Content.Shared/Damage/Container/DamageContainerPrototype.cs

```diff
@@ -73,13 +73,13 @@ void ISerializationHooks.AfterDeserialization()
         {
             _prototypeManager = IoCManager.Resolve<IPrototypeManager>();
 
-            // If all damge types are ALL supported, add all of them.
+            // Are all damage types are supported?
```

Comment:
```suggestion
```
The if provides enough info I think, so the comment isn't needed, but if you want to keep the comment, then there's an extra 'are' in this sentence.